lib/generated/r.dart

```diff
@@ -0,0 +1,24 @@
+class R {
```

Comment:
Can we rename this file to constants/assets.dart ?

Reply:
I followed the default naming convention from android.

> Android R.java is an auto-generated file by aapt (Android Asset Packaging Tool) that contains resource IDs for all the resources of res/ directory.
[ref](https://www.javatpoint.com/R-java-file-in-android)


and it's a generated file so the generated package would be a better place for it as it shouldn't be edited by hand.